scripts/run-postInstall.js

```diff
@@ -3,7 +3,7 @@ const { execSync } = require('child_process')
 const executionEnv = process.env.CI ? 'ci' : 'local'
 
 const postInstallCommands = {
-  local: 'patch-package && yarn-deduplicate --strategy=highest && yarn clean && gulp postinstall && yarn build && yarn build-v8-snapshot-dev',
+  local: 'patch-package && yarn-deduplicate --strategy=highest && yarn clean && gulp postinstall && yarn build && DEBUG=cypress:snapgen:info yarn build-v8-snapshot-dev',
```

Comment:
Do we intentionally want to keep the debug logs on for snapgen when we do a post install?
```suggestion
  local: 'patch-package && yarn-deduplicate --strategy=highest && yarn clean && gulp postinstall && yarn build && yarn build-v8-snapshot-dev',
```

Reply:
I added this so that there would be some level of progress as this was being generated. I don't want people to think that things are hanging for some reason